llvm/lib/TargetParser/RISCVTargetParser.cpp

```diff
@@ -95,5 +96,31 @@ void fillValidTuneCPUArchList(SmallVectorImpl<StringRef> &Values, bool IsRV64) {
 #include "llvm/TargetParser/RISCVTargetParserDef.inc"
 }
 
+void getFeaturesForCPU(StringRef CPU,
```

Comment:
This code is dead in tree. We should have a comment indicating why it shouldn't be deleted.

Reply:
Good point, thanks!